README.adoc

```diff
@@ -111,13 +160,32 @@ After you see the following message in both command-line sessions, both of your
 The defaultServer server is ready to run a smarter planet.
 ----
 
+// static guide instructions:
+ifndef::cloud-hosted[]
 Make sure that your Jaeger server is running and point your browser to the {inv-url}/localhost[{inv-url}/localhost^] URL.
 When you visit this endpoint, you make two GET HTTP requests, one to the `system` service and one to the `inventory`
 service. Both of these requests are configured to be traced, so a new trace is recorded in Jaeger.
 
 To view the traces, go to the {jaeger-url}[{jaeger-url}^] URL.
 You can view the traces for the inventory or system services under the **Search** tab in the upper left of the page.
 Select the services in the **Select a service** menu and click the **Find Traces** button at the end of the section.
+endif::[]
+
+// cloud-hosted guide instructions:
+ifdef::cloud-hosted[]
+Open another command-line session and run the following curl command from the terminal:
+```
+curl http://localhost:9081/inventory/systems/localhost
+```
+{: codeblock}
+
+When you visit this endpoint, you make two GET HTTP requests, one to the **system** service and one to the **inventory**
+service. Both of these requests are configured to be traced, so a new trace is recorded in Jaeger.
+
+To view the traces, go to the **`https://accountname-16686.theiadocker-4.proxy.cognitiveclass.ai`** URL.
+You can view the traces for the inventory or system services under the **Search** tab in the upper left of the page.
```

Comment:
"under the **Search** tab in the upper left of the page."
For accessibility compliance, avoid directional language- "upper left of the page"
If there is not another clear way to give an indication of where to look for the search tab, i think the following is ok:
"You can view the traces for the inventory or system services under the **Search** tab."

Reply:
updated